pkg/apis/machine/v1beta1/machine_types.go

```diff
@@ -30,6 +33,10 @@ const (
 
 	// MachineClusterLabelName is the label set on machines linked to a cluster.
 	MachineClusterLabelName = "cluster.k8s.io/cluster-name"
+
+	// MachineClusterIDLabel is the label that a machine must have to identify the
+	// cluster to which it belongs.
+	MachineClusterIDLabel = "machine.openshift.io/cluster-api-cluster"
```

Comment:
Should all the instances of `machine.openshift.io/cluster-api-cluster` in MAO be referred by this const?
Is there a reason for not prefering `machine.openshift.io/cluster-id` over `machine.openshift.io/cluster-api-cluster` ?

Reply:
The label is already generated by the installer so I am just copy-pasting it.